src/analyses/useAfterFree.ml

```diff
@@ -24,83 +24,110 @@ struct
 
   (* HELPER FUNCTIONS *)
 
+  let set_global_svcomp_var is_double_free =
+    if is_double_free then
+      AnalysisState.svcomp_may_invalid_free := true
+    else
+      AnalysisState.svcomp_may_invalid_deref := true
+
```

Comment:
This is also changed in the branch for the SV-COMP "Memory Safety" benchmarks and will be added to Goblint in a separate PR